weave/pantry.py

```diff
@@ -1,8 +1,9 @@
-"""This class builds the user-facing Index class. It pulls from the _Index 
+"""This class builds the user-facing Index class. It pulls from the _Index
 class which uses Pandas as it's backend to build and interface with the on disk
 Index baskets.
 """
 
+import copy
```

Comment:
Why do we need to deep copy the pantry? The mongo loader class only uses it to reference the pantry path and some other basic info.

Reply:
I didn't if any of the pantry's attribute were going to be modified.

Reply:
removed